octodns/record/ds.py

```diff
@@ -42,25 +42,25 @@ def validate(cls, data, _type):
         reasons = []
         for value in data:
             try:
-                int(value['flags'])
+                int(value['key_tag'])
```

Comment:
Unfortunately we'll need to fall back to looking for the previous names for backwards compatibility in case anyone has defined existing records with the original names. 

Cleanest way I can think of to update this section is something like:

```suggestion
                # key_tag is correct, but we previously released this with
                # `flags` so we need to check box
                int(value.get('flags', None) or value['key_tag'])
```

We'll need to do the same thing for all of them.

Reply:
Problem is that we have the algorithm field in both sets of fields (and for different fields) so I have done it a bit differently.